### ISSUE TITLE
America/Los_Angeles miss resolves to America/Dawson

### ISSUE THREAD

**Issue by otasman** [ndom91/react-timezone-select#34]
"America/Los_Angeles" resolves to America/Dawson when given as value instead of America/Tijuana or a different Pacific Time Timezone

**Comment by otasman** [comment]
Did you have any luck with this? All that was needed to recreate the issue was to set the initial state of the selected time zone as in the example below.
```typescript
    const [selectedTimezone, setSelectedTimezone] = useState("America/Los_Angeles")
```

![image](https://user-images.githubusercontent.com/10455596/124651462-f303a700-de68-11eb-998f-853e3d89d84f.png)

**Comment by ndom91** [comment]
@otasman thanks for reminding me, I just published a new version to npm (`1.0.5`) which adds a dedicated timezone for US Pacific Time under `America/Los_Angeles` as it was previously grouped in under `America/Tijuana`. Tijuana now has its own and all of America PDT is now under LA.

Im going to close this issue now, if you still have issues with this feel free to reopen!
